ui/app/routes/application.js

```diff
@@ -56,7 +56,7 @@ export default class ApplicationRoute extends Route {
         this.controllerFor('application').set('error', e);
       }
 
-      const fetchSelfTokenAndPolicies = this.get(
+      const fetchSelfTokenAndPolicies = await this.get(
```

Comment:
Fetching policies is now a potentially multi-step process (if they have roles, its policies are subsequently fetched), so we use `await`.